content/working_with/official_plugins/Orchestration/terraform.md

```diff
@@ -122,7 +122,62 @@ This refers to a Terraform module.
 
     * flags_override: The plugin has its own internal logic for appending flags to the tflint command.  However, if you wish to add or modify flags, configure here.  For example, "{'loglevel': 'debug'}", becomes "--loglevel=debug".
     * env: Additional env vars for duration of tflint executions,
-
+    * enable: boolean, In order for it to work, must mark True.
+    
+  * tfsec_config:  tfsec is a static analysis security scanner for your Terraform code.
+    * installation_source: The URL to download the tfsec binary from, e.g. 'https://github.com/aquasecurity/tfsec/releases/download/v1.1.3/tfsec-linux-amd64'.
+    * executable_path:  If the binary is located on the file system, this is the path on the file system, e.g. /usr/local/bin/tfsec.
+    * config: tags, as valid JSON (NOT HCL)
+    * flags_override: 'tfsec can by run with no arguments and will act on the current folder.
+          For a richer experience, there are many additional command line arguments that you can make use of.
+          For example: [ "debug", "run-statistics"] (without --).
+          e.g 'https://aquasecurity.github.io/tfsec/v1.2.1/getting-started/usage/'
+    * enable: boolean, In order for it to work, must mark True.
+
+    config.yml
+
+    ```yaml      
+    tfsec_config:
+        config:
+            exclude: 
+              - 'aws-vpc-add-description-to-security-group-rule'
+              - 'aws-vpc-no-public-egress-sgr' 
+              - 'aws-vpc-no-public-ingress-sgr'
+        flags_override: []
+        enable: True
+    ```
+     or config.json:
+
+    ```yaml      
+    tfsec_config:
+        config: { 
+                    "exclude" : 
+                    ['aws-vpc-add-description-to-security-group-rule','aws-vpc-no-public-egress-sgr','aws-vpc-no-public-ingress-sgr']
+                }
+        flags_override: []
+        enable: True
+    ```
+  * terratag_config: 
+    * installation_source: The URL to download the terratag binary from, e.g. 'https://github.com/env0/terratag/releases/download/v0.1.35/terratag_0.1.35_linux_amd64.tar.gz'.
+    * executable_path:  If the binary is located on the file system, this is the path on the file system, e.g. /usr/local/bin/terratag.
+    * tags: tags, as valid JSON (NOT HCL)
+    * flags_override: 
+      * -dir=<path> - defaults to '.'. Sets the terraform folder to tag .tf files in.
+      * -skipTerratagFiles=false - Dont skip processing *.terratag.tf files (when running terratag a second time for the same directory).
+      * -verbose=true - Turn on verbose logging.
+      * -rename=false - Instead of replacing files named <basename>.tf with <basename>.terratag.tf, keep the original filename.
+      * -filter=<regular expression> - defaults to .*. Only apply tags to the resource types matched by the regular expression.
```

Comment:
Should we be providing the examples with the dash (`-`) in the key names? Or are these for new list items in markdown language? If so add space after dash.